implementations/rust/ockam/ockam_api/src/cloud/project.rs

```diff
@@ -355,7 +355,8 @@ mod tests {
                     name: String::arbitrary(g).into(),
                     space_name: String::arbitrary(g).into(),
                     services: vec![String::arbitrary(g).into(), String::arbitrary(g).into()],
-                    access_route: String::arbitrary(g).into(),
+                    access_route: MultiAddr::from_str(&format!("node/{}", String::arbitrary(g)))
+                        .unwrap(),
```

Comment:
The string here can not be arbitrary, as the '/' has special meaning when parsing textual multi-addresses.

```suggestion
                    access_route: {
                        let mut ma = MultiAddr::default();
                        ma.push_back(ockam_multiaddr::proto::Node::new(String::arbitrary(g)).unwrap();
                        ma
                    },
```